### ISSUE TITLE
show axiom annotations on text definitions

### ISSUE THREAD

**Issue by cmungall** [EBISPOT/OLS#190]
e.g

https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0024265

it would be great to see the provenance of the definition

**Comment by simonjupp** [comment]
Yep, if you look in the JSON we already have extracted the definition_citation into a more structured object, so this is simply a case of updating the webpages to render this.
